src/core/server/base.js

```diff
@@ -378,6 +379,8 @@ export function runServer({
   const host = config.get('serverHost');
   const appName = config.get('appName');
 
+  const useHttps = process.env.USE_HTTPS;
```

Comment:
We should make it more obvious that this is for development only:

```js
const useHttpsForDev = process.env.USE_HTTPS_FOR_DEV;
```

Reply:
Good point.